src/Runner.php

```diff
@@ -80,18 +80,15 @@ public function execute($input = null)
         register_shutdown_function(array($this, 'shutdown'));
         set_error_handler(array($this, 'handleError'));
 
-        // If we were not provided with a container, then create one
+        // If we were not provided a container, then create one
         if (!Config::hasContainer()) {
             $input = $this->prepareInput($input ? $input : $_SERVER['argv']);
             $container = Config::createContainer($input);
             Config::setContainer($container);
-
-            // Note: this freezes our container, preventing us from adding any further
-            // services to it.
-            $container->compile();
```

Comment:
League doesn't support compiling containers, so it was easy for me to make the right choice here. :)